cristin-person/src/main/java/no/unit/nva/cristin/person/country/ListCountriesApiClient.java

```diff
@@ -0,0 +1,66 @@
+package no.unit.nva.cristin.person.country;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyMap;
+import static no.unit.nva.client.HttpClientProvider.defaultHttpClient;
+import static no.unit.nva.utils.UriUtils.createCristinQueryUri;
+import static no.unit.nva.utils.UriUtils.getNvaApiUri;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import no.unit.nva.cristin.common.client.ApiClient;
+import no.unit.nva.cristin.person.model.cristin.CristinCountry;
+import no.unit.nva.cristin.person.model.nva.Countries;
+import no.unit.nva.cristin.person.model.nva.Country;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadGatewayException;
+
+public class ListCountriesApiClient extends ApiClient {
+
+    public static final String COUNTRY_PATH = "country";
+    public static final URI COUNTRY_ID_URI = getNvaApiUri(COUNTRY_PATH);
+    public static final String CRISTIN_COUNTRIES_PATH = "countries";
+    public static final URI COUNTRY_CONTEXT_JSON = URI.create("https://example.org/country-context.json");
+
+    public ListCountriesApiClient() {
+        this(defaultHttpClient());
+    }
+
+    public ListCountriesApiClient(HttpClient httpClient) {
+        super(httpClient);
+    }
+
+    public Countries executeRequest() throws ApiGatewayException {
+        Map<String, String> params = emptyMap();
+        var queryUri = createCristinQueryUri(params, CRISTIN_COUNTRIES_PATH);
+        var start = System.currentTimeMillis();
+        var response = queryUpstream(queryUri);
+        var countries = getCountries(response);
+        var totalProcessingTime = calculateProcessingTime(start, System.currentTimeMillis());
```

Comment:
im considering removing processingtime as I am caching the lambda response with cloudfront

Reply:
Maybe only interesting for debug, so probably — we have other ways of getting this info now and as you point out the cache makes the info stale

Reply:
i will remove this field